src/main/java/org/candlepin/model/OwnerProductCurator.java

```diff
@@ -951,7 +959,8 @@ private void updateOwnerProductActivationKeys(Owner owner, Map<String, String> p
             sql = "UPDATE cp2_activation_key_products SET product_uuid = :updated " +
                 "WHERE key_id = :key_id AND product_uuid = :current";
 
-            query = entityManager.createNativeQuery(sql);
+            query = entityManager.createNativeQuery(sql)
```

Comment:
The `updateOwnerProductActivationKeys` method is dead and uncalled. We can just remove it entirely rather than update it.

Reply:
:+1: